client/src/Components/overview/MainOverview.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+import DefaultImageGallery from './DefaultImageGallery.jsx';
+import getStyles from './helpers/getStyles.jsx';
+import ProductSelection from './ProductSelection.jsx';
+
+const MainOverview = () => {
+  const [isExpanded, setExpanded] = useState(false);
+  const [styleIdx, setStyleIdx] = useState(0);
+  const product = useSelector((state) => state.product) || {};
+  console.log('product: ', product);
```

Comment:
// Left over console log
```suggestion
  console.log('product: ', product);
```